### PR TITLE
Design feedback update

### PR DESCRIPTION
**Pull request by traceykthompson**
Updated The Design Review Best Practices and Giving + Receiving feedback guides. 
Added a TOC to each and small content changes for clarity. 
Also added a cross-link from G+R Feedback to Design Review

### CODE REVIEW

File: product_design/how_we_write_user_stories.md [hunk 1]
```diff
@@ -1,105 +1,89 @@
...
+## The role of Epics and User Stories in our work 
 
-**Stories focus conversation.** A story allows the team to make multiple confirmations of the task throughout the development process. This begins with agreement on the task to be done, and later, how it will pass acceptance testing.
+Epics and User Stories are tools that focus our conversation around what to build and why. At the highest level, they embody the things that make our product worth building — who it’s for and what that person can do with the product to achieve their goals.
 
-## What is needed before we write user stories?
+- Epics describe the vision and the outcomes we are striving to achieve and how we intend to measure success. 
+- User stories generate empathy and alignment around a discrete effort and how that work will benefit the user. Writing user stories helps us come to an agreement on what needs to be done and what criteria we will use to tell when the work is complete.
```
Comment (Shiun) [inline product_design/how_we_write_user_stories.md:11]: Should we mention that Epics are composed of a set of stories?  Epics are high level visions or large slices of functionality?  And stories are individual discrete pieces of work in the epic?  (need better word smithing here)

File: assets/images/structuring_tickets/structure-speclet1.png
Comment (Shiun) [inline assets/images/structuring_tickets/structure-speclet1.png:1]: Contenders in this context == candidates?  It seems like these two terms are interchanged in the pngs.

File: product_design/structuring_epics_stories_bugs_and_tasks.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+![Initiatives inform Epics which then inform our stories, tasks, and bugs](/assets/images/structuring_tickets/structure-overview.png)
+
+<br>
+
+---
+## Initiatives
+Initiatives are a collection of epics that drive towards a common goal (for example: “expand in to Australia” or “launch a new product that decreases costs by 7% this year”). An initiative is broader than a single epic, and may span multiple teams and multiple quarters.
+
+## Epics
+Epics convey the overall big picture priorities. They contain a group of stories (on a particular theme) that are too big to be described by a single user story. Epics are written concisely and in a way that is simple to understand.
```
Comment (Shiun) [inline product_design/structuring_epics_stories_bugs_and_tasks.md:18]: "They contain a group of stories (on a particular theme) that are too big to be described by a single user story. Epics are written concisely and in a way that is simple to understand."

OK it looks like we mention it here.  (re: Epics = group of stories)

File: product_design/how_we_write_user_stories.md (hunk 1 above)
Comment (traceykthompson) [reply product_design/how_we_write_user_stories.md:11]: @katebennet would you be able to address @Shiun's content questions?

**Review by Shiun — APPROVED** [review]
This looks good.  The content of the speclet png probably don't need to be address since it's not relevant to the guide.  Especially since the example speclets are referenced for their structure, not content.

Maybe a re-interation of Epics being composed of stories would be a good edition in the definition of Epics/Stories.

Otherwise, looks great!

**Pull request merged**